homeassistant/components/sleepiq.py

```diff
@@ -39,7 +39,7 @@
 DATA = None
 
 CONFIG_SCHEMA = vol.Schema({
-    DOMAIN: vol.Schema({
+    vol.Required(DOMAIN): vol.Schema({
```

Comment:
How could the component be setup without the domain in the config? From another component?